docs/src/indexing-flow.md

```diff
@@ -79,6 +79,32 @@ add_filter( 'algolia_should_index_user', 'filter_user', 10, 2 );
 In the above example, User with ID 1 would never get indexed. This example is trivial, but at least gives you a quick overview on how to filter your items to index.
 Also note how we return `false` early on if the decision has already been taken to not index the content.
 
+**Another example to exclude posts from the searchable_posts index where the noindex option from the Yoast SEO plugin is set to "noindex"**
+```php
```

Comment:
Do you mind adding a small warning here to not confuse user not using the YOAST plugin?

```
<div class="alert alert-warning">The following plugin assumes you are using the YOAST SEO plugin for WordPress</div>
```